perun-core/src/main/java/cz/metacentrum/perun/core/provisioning/GroupsHashedDataGenerator.java

```diff
@@ -68,6 +68,7 @@
 	private final Facility facility;
```

Comment:
Shouldn't ``hashes: [...hashes...]`` be removed also from this class javadoc? It was removed from services manager methods API docs.

Reply:
Yes, I have removed it.